{
  "Faults": [
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-14T08:30:00.000+0000",
      "latitude": 51.50575824001529,
      "estimatedRestoration": "2022-03-18T16:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '206219'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/206219",
      "reference": "206219",
      "name": "SN14 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SN14 6BJ",
        "SN14 6DG",
        "SN14 6HE",
        "SN14 6JE",
        "SN14 6JF",
        "SN14 6JG",
        "SN14 6JH",
        "SN14 6JJ",
        "SN14 6JL",
        "SN14 6JN",
        "SN14 6JP",
        "SN14 6JQ",
        "SN14 6JW",
        "SN14 6LD",
        "SN14 6LE",
        "SN14 6NH",
        "SN14 6PS",
        "SN14 6PU",
        "SN14 6QL",
        "SN14 6RF",
        "SN14 6RL",
        "SN14 7NA",
        "SN14 8RS"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -2.160254865602929,
          51.50575824001529
        ],
        "type": "Point"
      },
      "UUID": "3fa8bbb9-cfb1-487c-b7a4-b25d051dd09d",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.160254865602929,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-14T09:00:00.000+0000",
      "latitude": 51.44762043180477,
      "estimatedRestoration": "2022-03-18T15:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '206573'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/206573",
      "reference": "206573",
      "name": "SN11 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SN11 8RR"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.9897779460009772,
          51.44762043180477
        ],
        "type": "Point"
      },
      "UUID": "389297d4-a19b-4f4b-a077-a888c1728445",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.9897779460009772,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-14T21:18:00.000+0000",
      "latitude": 51.54700976890266,
      "estimatedRestoration": "2022-03-15T09:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JQ0445'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JQ0445",
      "reference": "JQ0445",
      "engineerOnSiteTime": "2022-03-14T22:12:00.000+0000",
      "name": "SN4 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SN4 0EF",
        "SN4 0EH",
        "SN4 0EJ",
        "SN4 0EL",
        "SN4 0EN",
        "SN4 0ES",
        "SN4 0EW",
        "SN4 0FA",
        "SN4 0FQ"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.6976153821250524,
          51.54700976890266
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-03-14T22:15:00.000+0000",
      "UUID": "2f5a68f7-23b4-43e1-b9a8-56527bb22b52",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.6976153821250524,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-14T21:12:00.000+0000",
      "latitude": 51.569670657508176,
      "estimatedRestoration": "2022-03-15T09:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JQ0446'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JQ0446",
      "reference": "JQ0446",
      "name": "SL7 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SL7 1LU",
        "SL7 1LW",
        "SL7 1QF",
        "SL7 1QH",
        "SL7 1QQ",
        "SL7 1QT",
        "SL7 1QZ",
        "SL7 1SZ",
        "SL7 2AH"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.764648536494211,
          51.569670657508176
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-03-14T22:00:00.000+0000",
      "UUID": "1817e3f5-58e4-4e6b-8f98-c9afb5884037",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "evChargersAffected": [
        "089c520a-d228-4372-b07b-4fd92dfa3121"
      ],
      "longitude": -0.764648536494211,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-15T06:31:00.000+0000",
      "latitude": 51.560465226221744,
      "estimatedRestoration": "2022-03-15T09:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JQ0488'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JQ0488",
      "reference": "JQ0488",
      "engineerOnSiteTime": "2022-03-15T05:20:00.000+0000",
      "name": "SN1 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SN1 5AR",
        "SN1 5BG",
        "SN1 5DT",
        "SN1 5EE",
        "SN1 5JA",
        "SN1 5JD",
        "SN1 5JE",
        "SN1 5JP"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.7902808192152604,
          51.560465226221744
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-03-15T05:30:00.000+0000",
      "UUID": "e2ac70a9-dbd0-4843-8010-39f1ecbd447a",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.7902808192152604,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-15T06:46:00.000+0000",
      "latitude": 51.23387425522813,
      "estimatedRestoration": "2022-03-15T10:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JQ0503'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JQ0503",
      "reference": "JQ0503",
      "name": "RG23 & RG25 Areas",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "RG23 7ED",
        "RG23 7EH",
        "RG23 7EN",
        "RG23 7EP",
        "RG23 7ER",
        "RG23 7GE",
        "RG23 7HA",
        "RG23 7HP",
        "RG23 7HR",
        "RG23 7JJ",
        "RG23 7JT",
        "RG23 7LE",
        "RG23 7LF",
        "RG23 7LG",
        "RG23 7LH",
        "RG23 7LJ",
        "RG23 7LQ",
        "RG23 7LX",
        "RG23 7LY",
        "RG23 7NA",
        "RG23 7ND",
        "RG23 7NE",
        "RG23 7NH",
        "RG23 7NJ",
        "RG23 7NQ",
        "RG23 7TW",
        "RG24 8QZ",
        "RG25",
        "RG25 2HG",
        "RG25 3AB",
        "RG25 3AD",
        "RG25 3AH",
        "RG25 3AJ",
        "RG25 3AQ",
        "RG25 3AR",
        "RG25 3BF",
        "RG25 3BJ",
        "RG25 3BL",
        "RG25 3BN",
        "RG25 3BP",
        "RG25 3BS",
        "RG25 3BW",
        "RG25 3BY",
        "RG25 3DH",
        "RG25 3DN",
        "RG25 3DR",
        "RG25 3DS",
        "RG25 3DW",
        "RG25 3EA",
        "RG25 3EB",
        "RG25 3ED",
        "RG25 3HY",
        "RG25 3HZ",
        "RG25 3LG",
        "RG25 3LJ",
        "RG25 3LQ",
        "RG25 3LY",
        "RG25 3LZ",
        "RG25 3NB",
        "RG25 3ND",
        "RG25 3NE",
        "RG25 3NF",
        "RG25 3NG",
        "RG25 3NH",
        "RG25 3NQ",
        "RG25 3NT",
        "RG25 3NX",
        "RG25 3NZ",
        "RG25 3PB",
        "RG25 3PF",
        "RG25 3PH",
        "RG25 3TG",
        "RG28 7PD",
        "RG28 7PE",
        "RG28 7PF",
        "RG28 7PG",
        "RG28 7RU",
        "SO21 3AN",
        "SO21 3AP",
        "SO21 3AR",
        "SO21 3BA",
        "SO21 3BB",
        "SO21 3BD",
        "SO21 3BE",
        "SO21 3BG",
        "SO21 3BW",
        "SO21 3SP"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.23149348085256,
          51.23387425522813
        ],
        "type": "Point"
      },
      "UUID": "84ad7356-3d2b-4465-acc2-a3650f27e428",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "evChargersAffected": [
        "7b7ea59c-f063-436e-95e3-0e5060eaae79"
      ],
      "longitude": -1.23149348085256,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-14T10:00:00.000+0000",
      "latitude": 59.11693007695559,
      "estimatedRestoration": "2022-03-21T19:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '206022'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/206022",
      "reference": "206022",
      "name": "KW17 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "KW17 2PJ"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -3.119189129637787,
          59.11693007695559
        ],
        "type": "Point"
      },
      "UUID": "12c3f463-535e-47d1-a23a-cd851a71c99f",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -3.119189129637787,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-15T04:45:00.000+0000",
      "latitude": 56.84389800199144,
      "estimatedRestoration": "2022-03-15T09:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JQ0448'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JQ0448",
      "reference": "JQ0448",
      "name": "PH33 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "PH33 7JG",
        "PH33 7JH",
        "PH33 7JJ",
        "PH33 7JL",
        "PH33 7JN",
        "PH33 7JR",
        "PH33 7JW"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -5.126114402657437,
          56.84389800199144
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-03-14T22:40:00.000+0000",
      "UUID": "a56ac9d2-aaea-4b3b-bb4b-ff3ab1ba9833",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -5.126114402657437,
      "resolved": false
    }
  ]
}
